frame.go

```diff
@@ -32,7 +32,7 @@ import (
 
 // Default frame settings.
 const (
-	DefaultRowLabel       = "rowID"
+	DefaultRowLabel       = "row"
```

Comment:
shouldn't this setting not exist anymore?

Reply:
Well, the constant needs to be defined somewhere (unless we want the string "row" scattered throughout the code). Is it that the name of the constant is confusing?  Perhaps changing it to `RowLabel` or something else makes more sense.

Reply:
I could also move those constants to `executor.go` since that's the only place they're used anymore.

Reply:
+1 on name change and moving

Reply:
moved and renamed in e3cd03d902174524de1082cff72c609b9342b8f6